src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlAuthenticationProviderManager.cs

```diff
@@ -28,8 +28,26 @@ internal partial class SqlAuthenticationProviderManager
         private readonly SqlClientLogger _sqlAuthLogger = new SqlClientLogger();
         private readonly string _applicationClientId = ActiveDirectoryAuthentication.AdoClientId;
 
-        public static readonly SqlAuthenticationProviderManager Instance;
+        public static readonly SqlAuthenticationProviderManager s_instance;
```

Comment:
NIT: this is a public field and you can keep the uppercase naming convention here.

Reply:
The class is internal, so doesn't really affect any public surface.
I just followed recommendations from VS.